config/boards/hinlink-h28k.conf

```diff
@@ -0,0 +1,21 @@
+# Rockchip RK3568 quad core 1-8GB SoC GBe eMMC USB3
```

Comment:
is this an RK3528 or RK3568?

Reply:
Modified，ths！

Reply:
Rockchip new cpu RK3528 :https://www.cnx-software.com/news/rockchip/